lib/engine/game/g_1880/game.rb

```diff
@@ -428,6 +468,16 @@ def init_minors
           game_minors.map { |minor| G1880::Minor.new(**minor) }
         end
 
+        def init_train_handler
+          trains = game_trains.flat_map do |train|
+            Array.new((train[:num] || num_trains(train))) do |index|
+              Train.new(**train, index: index)
+            end
+          end
+
+          G1880::Depot.new(trains, self)
```

Comment:
Instead of reimplementing the method, follow what I did with CORPORATION_CLASS in base game

Reply:
done!